enhancements/console/internationalization.md

```diff
@@ -110,6 +110,14 @@ The desired fallback experience if not all pieces/parts of the UI are translated
 
 We will also modify our existing file conversion tools to handle translation files spread across multiple packages so we can easily export/import translation files.
 
+Translation of the login page will be handled by [oauth-server](https://github.com/openshift/oauth-server), which will check firstly the request URL for `ui_locales` query. If the query is missing, `oauth-server` will check for request `Accept-Language` header. If both the query and header are missing in the request, `oauth-server` will default to English language for localization.
+
+[oauth-server](https://github.com/openshift/oauth-server) contains the default OKD login templates. Custom templates are stored in the [oauth-templates](https://github.com/openshift/oauth-templates) repository and will also need to be updated for the other brands like the OCP and OpenShift Dedicated login pages.
```

Comment:
do we need a bump of oauth-server when the templates change? I assume the console team will do those?

Reply:
yes, if any of the templates change we will make needed changes.
Will incorporate that into the enhancement

Reply:
In case of any changes in these repositories, Console-Team will be responsible for syncing those changes between them.

Reply:
updated text